spec/02-integration/02-cmd/02-start_stop_spec.lua

```diff
@@ -18,6 +18,60 @@ describe("kong start/stop #" .. strategy, function()
     helpers.clean_prefix()
   end)
 
+  it("with #referenced values [fails]", function()
```

Comment:
removed tag, reads like English
```suggestion
  it("fails with referenced values that are not initialized", function()
```